antlr/src/main/antlr/FSHLexer.g4

```diff
@@ -120,3 +120,9 @@ LINE_COMMENT:       '//' ~[\r\n]* [\r\n] -> skip;
 mode RULESET_OR_INSERT;
 PARAM_RULESET_REFERENCE:      WS* NONWS+ (WS* ('(' ('\\)' | '\\\\' | ~[)])+ ')')) -> popMode;
 RULESET_REFERENCE:            WS* NONWS+ -> popMode;
+
+mode ALIAS;
+ALIAS_EQUAL:        '=';
+ALIAS_SEQUENCE:     NONWS+;
+NEWLINE:            [\r\n] -> popMode;
+ALIAS_WHITESPACE:   WS -> channel(HIDDEN);
```

Comment:
I don't see where the `ALIAS_WHITESPACE` keyword is used. Is there a reason to have it defined so we can use it in the future?

Reply:
It serves the same purpose as the `WHITESPACE` token in the other mode. So it consumes whitespace and sends it to the hidden channel to be ignored.

Reply:
No longer relevant as of https://github.com/FHIR/sushi/pull/880/commits/c2b7e1413b6cfc3b0096eb0a61dfdfc6f670e421.